### PR TITLE
Fix windows nightlies url

### LINKED ISSUE #10: Windows nightlies are now zips
As a reminder, newer nightly versions of windows are now distributed as a zip. Will probably cause some issues here: https://github.com/lix-pm/switchx/blob/master/src/switchx/Switchx.hx#L43

### PR DESCRIPTION
**Pull request by benmerckx**
Fixes #10

### CODE REVIEW

File: src/switchx/Switchx.hx [hunk 1]
```diff
@@ -216,15 +221,22 @@ class Switchx {
       root.delete();
   }
       
-  public function download(version:ResolvedVersion, options:{ force: Bool }):Promise<Bool>
+  public function download(version:ResolvedVersion, options:{ force: Bool }):Promise<Bool> {
+    
+    inline function downloadArchive(url, peel, into)
```
Comment (back2dos) [inline src/switchx/Switchx.hx:226]: Fun fact: [`Download.archive`](https://github.com/lix-pm/switchx/blob/master/src/switchx/Download.hx#L36) does pretty much the same thing as `downloadArchive` :P

Comment (benmerckx) [reply src/switchx/Switchx.hx:226]: Right! I'd like to put that change in your hands and bug you for a release while I'm at it :)

Comment (back2dos) [reply src/switchx/Switchx.hx:226]: @benmerckx done ;)

Comment (benmerckx) [reply src/switchx/Switchx.hx:226]: You're the best, I felt like that could go either way! :)

**Pull request merged**